src/popper/commands/cmd_version.py

```diff
@@ -7,8 +7,7 @@
 @click.command("version", short_help="Show version of Popper and exit.")
 @pass_context
 def cli(ctx):
-    """Displays version of Popper and exit.
-    """
+    """Displays version of Popper and exit."""
```

Comment:
@maruthgoyal Are these changes done by black? Same for the other files. I guess not. You can just make the changes to the `cmd_sh.py` file and make the PR.

Reply:
Hey Jayjeet, apologies I believe those changes were made by the run of the previous version of black. I have restored the file's respective states to that of the latest commit on master.

Reply:
No worries. Thanks !